editor/src/core/shared/element-path-tree.ts

```diff
@@ -133,6 +134,69 @@ export function reorderTree(trees: ElementPathTrees, metadata: ElementInstanceMe
   }
 }
 
+function maybeReorderDynamicChildren(
+  children: ElementPathTree[],
+  metadata: ElementInstanceMetadataMap,
+): ElementPathTree[] {
+  function keepOnlyUnique(path: ElementPath, index: number, array: ElementPath[]) {
+    return array.indexOf(path) === index
```

Comment:
Surely it needs to check more than the first index it finds?
```suggestion
    return array.indexOf(path) === index && array.lastIndexOf(path) === index
```

Reply:
🤔  I don't think it needs to, it needs to trim uniques from the head of the array

Reply:
Oh yes, my mistake, this will filter out subsequent instances that match earlier ones!